Meetbot_stats

bee2502 opened the pull request:
A few lines in script have been tweaked for file writing. Sample Results files(monthwise.txt , weekwise.txt , daywise.txt ) are also updated in the /scripts

bee2502 committed:
Update meetbot_stats.py

Few lines have been tweaked. Computation Time : ~3 hrs( Monthwise ~ 30 mins , Weekwise ~ 60 mins , Daywise ~ 90 mins)

bee2502 committed:
Create monthwise.txt

Monthwise results

bee2502 committed:
Create weekwise.txt

weekwise results

bee2502 committed:
Create daywise.txt

daywise results

decause commented:
If you're going to list the dates going back in time, it may be a good idea to output and add the datetime that the script was run at, so that the data has some context?

@ralphbean since you've written the orig, how about a quick peek?

ralphbean commented:
> it may be a good idea to output and add the datetime that the script was run at, so that the data has some context?

Sure.  That would make sense.

pypingou commented:
hm, are we mixing tabs and spaces in this file here?

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Updated indents

ralphbean commented:
Friendly tip, @bee2502:  use more specific messages and what you changed in your commit messages.  Otherwise, it is hard to look at the history of commits here (and elsewhere) and quickly know what changed, and why.

pypingou commented on scripts/meetbot_stats.py:
```diff
@@ -15,12 +15,13 @@ def get_count(start, finish, query):
-        'https://apps.fedoraproject.org/datagrepper/raw',
-        params=query,
+    'https://apps.fedoraproject.org/datagrepper/raw',
+    params=query,
```
Missed one here :)

bee2502 committed:
Updated Indents and Time Stamped Values in Code; Added Monthwise TimeStamped Results

bee2502 committed:
Updated Indents and Time Stamped Values in Code; Added Monthwise Time Stamped Results

bee2502 commented:
@ralphbean , @decause : I have updated the indents and edited the code to provide timestamped values. I have also added a sample file for monthwise results "TimeStampedMonthwise.txt". Please look into it.

Other than that - @ralphbean 's earlier code as well as the current script take the current time as input and find daywise/weekwise/monthwise results. Since, the script runs serially, the initial starting time for queries for all topics of a particular type(say monthwise) is not same. (Please check out the last part of 'TimeStampedMonthwise.txt' ) and also some time period is omitted. Do you think we should fix this issue by defining a universal start time for all topics/types and updating the time period accordingly or is this issue negligible ?

@pypingou : Updated the code . Thanks :)

bee2502 committed:
Updated the code-applied fixed start time for all queries/topics

bee2502 committed:
Update meetbot_stats.py

bee2502 committed:
Delete monthwise.txt

Removed Non TimeStamped Sample Results

bee2502 committed:
Delete weekwise.txt

Removed Non Time Stamped Sample Results

bee2502 committed:
Delete daywise.txt

Removed Non timestamped sample results

ralphbean commented:
@decause any thoughts on this?  You get the final say in a merge, I think.

decause commented:
If you're +1, I'm +1.

:shipit:
On Oct 23, 2015 12:10 PM, "Ralph Bean" notifications@github.com wrote:

> @decause https://github.com/decause any thoughts on this? You get the
> final say in a merge, I think.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/fedora-infra/fedora-stats-tools/pull/4#issuecomment-150620661
> .

bee2502 commented:
@decause , @ralphbean  : If possible, can we wait to merge? I want to add some meetbot topics not mentioned in the docs : https://github.com/fedora-infra/supybot-fedmsg/commit/47339f5741192cb0a2f5b65277cbce0d7c529bed
Documentation : https://fedora-fedmsg.readthedocs.org/en/latest/topics.html#meetbot

bee2502 committed:
Updated New Topics + Results

bee2502 committed:
Updated New Topics+Indents

bee2502 committed:
Updated New Topics + Results.

bee2502 commented:
Updated the code and results with new topics not available in documentation :
https://github.com/fedora-infra/supybot-fedmsg/commit/47339f5741192cb0a2f5b65277cbce0d7c529bed

ralphbean commented:
LGTM.  :+1:
